book-example/src/format/config.md

```diff
@@ -198,6 +198,10 @@ The following configuration options are available:
   an icon link will be output in the menu bar of the book.
 - **git-repository-icon:** The FontAwesome icon class to use for the git
   repository link. Defaults to `fa-github`.
+- **redirect:** A subtable used for generating redirects when a page is moved.
+  The table contains key-value pairs where the key is the path to the moved
+  page, relative to the build directory and the value can be an ordinary URL
+  or the *absolute* path to another page in the book
```

Comment:
I'm a bit confused why this emphasizes an *absolute* path.  If there is a redirect `"foo/bar.html" = "relative.html"`, I believe that will redirect to `foo/relative.html` correctly, will it not?  Can you say more why it says absolute here?

Reply:
Reading through that again, I think I got my logic back-to-front when writing the docs.

We want the pages which were moved (the bit before `=`) to be specified using absolute paths so there's no ambiguity over which file inside the build dir we're talking about. (i.e. it starts with `/`, therefore it's relative to the renderer's build directory)

You're correct about the redirect target (the bit after `=`) needing no constraints. We're just substituting the value into `<meta http-equiv="refresh" content="0;URL='{{url}}'">` and the browser would take care of everything for us, so pretty much anything you'd use in a normal markdown link is fair game.